### ISSUE TITLE
2.5.7 Ziehbewegungen: Auch bei nativen Schiebereglern?

### ISSUE THREAD

**Issue by jkphl** [BIK-BITV/BIK-Web-Test#448]
Die Prüfschrittbeschreibung enthält:

> Bei Schiebereglern gibt es die Möglichkeit numerischer Eingaben des Wertes über die (virtuelle) Tastatur.

Es wird dabei nicht spezifiziert, ob sich diese Aussage nur auf benutzerdefiniert konstruierte Schieberegler bezieht, oder auch einen nativen `<input type="range">` einschließt. Falls letzteres, bedeutet dies dann, dass *jeder Schieberegler* ein begleitendes Texteingabefeld benötigt? Falls nicht, dann sollte die Beschreibung entsprechend angepasst werden.

**Comment by detlevhfischer** [comment]
Ohne das jetzt näher recherchiert zu haben, vermute ich dass ein natives `input type="range"` 2.5.7 schon dadurch erfüllen würde, dass eine single pointer activation auf dem groove den thumb an diesen Ort versetzt. So wenigstens [hier](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range).

**Comment by jkphl** [comment]
@detlevhfischer Ja, so auch @sweckenmann's spontane Einschätzung. Dann sollte die Prüfschrittbeschreibung eine entsprechende Ergänzung erhalten.

**Comment by detlevhfischer** [comment]
@jkphl Nachdem ich mit den Prüfschritt noch mal angesehen habe, denke ich eigentlich nicht, dass eine Ergänzung nötig ist.
Die Bescrheibung listet ja drei unterschiedliche Wege zur Erfüllung auf:

- Aktivieren über einfache Zeigeraktion (z.B. Tippen / Klicken auf eine Stelle des Grooves wie bei nativen Slidern)
- Nach einfacher Zeigeraktion Einblenden von Elementen (Pfeile), die ein schrittweises Verschieben des Elements über einfache Zeigeraktion erlauben
- Alternative numerischer Eingabe des Wertes über die (virtuelle) Tastatur.

Vielliecht kann man den Text generalisieren: 
> Die Funktion kann auch über numerische Eingaben eines Wertes über die (virtuelle) Tastatur ausgeführt werden.

Damit ist der Slider nicht mehr im Vordergrund. Es könnte ja z.B. auch sein, dass nach Fokussierung eines ziehbaren Elements eines von mehreren möglichen durchnummerierten dragging targets über Eingabe der entsprechendn Zahl angesteuert wird. Wenn Du einen anderen Vorschlag hast, mach gern einen PR :)

**Comment by sweckenmann** [comment]
PR entsprechend Vorschlag von @detlevhfischer https://github.com/BIK-BITV/BIK-Web-Test/pull/483

**Comment by detlevhfischer** [comment]
Ich schließe das Issue mit Verweis auf PR #483
